model-optimizer/extensions/ops/elementwise.py

```diff
@@ -22,13 +23,49 @@ def override_data_type_of_constant(node: Node):
         # power value 2, or when replacing Neg operation with Mul with -1 as second input.
         in_node_0 = node.in_port(0).get_source().node
         in_node_1 = node.in_port(1).get_source().node
-        if in_node_0.op == 'Const':
-            convert_const_node_value_type(in_node_0, in_type_1)
-        elif in_node_1.op == 'Const':
-            convert_const_node_value_type(in_node_1, in_type_0)
+
+        if in_node_0.soft_get('op') != 'Const' and in_node_1.op != 'Const':
```

Comment:
nit: No need to use `soft_get` method for getting `op`. This attribute is always set for nodes.
But don't confuse with `type` attribute which may not be set in many many cases.

Reply:
reverted